news/2023/2023-11-16-osu-world-cup-2023-quarterfinals-recap.md

```diff
@@ -0,0 +1,51 @@
+
+Do you remember the crazy quarterfinals stage of the [2021 osu! World Cup](/wiki/Tournaments/OWC/2021#quarterfinals.1), including five tiebreakers? Not as many were needed the past weekend, but the amount of upsets and close battles made up for it and delivered one of the most entertaining stages in some years.
+
+As so many highlights needed to be included, the recap video is longer than usual. In case you just need a quick update, look below for the most outstanding matches.
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/ZkIb6p3_n6g" frameborder="0" allowfullscreen></iframe>
+</div>
+
+### ::{ flag=RO }:: Romania vs. ::{ flag=FR }:: France
+
+[**View the VOD here!**](https://www.twitch.tv/videos/1974857501?t=03h39m18s)
+
+Romania, the grim reaper of the lower bracket in the past few years, faced France for an elimination battle, and the match turned out to be neck and neck as expected. However, both teams struggled with winning their own picks, as six break points happened up to the scoreline of 4–4, including three cases of less than 80k points difference.
+
+At this point, this game could have gone either way. On the French HD2 pick ([BUTAOTOME - Street Journal](https://osu.ppy.sh/beatmapsets/2085341#osu/4368631)), ::{ flag=RO }:: [badeu](https://osu.ppy.sh/users/1473890) tried his very best to carry his team with a mind-boggling FC, but the rest of the Romania roster could not provide enough support. Consequently, a win on [yuikonnu - Taifuu Ikka? Risan](https://osu.ppy.sh/beatmapsets/2085338#osu/4368609) was desperately needed to force the tiebreaker. It looked good for the Eastern European team until the caster curse struck badeu — with ::{ flag=FR }:: [Fiaee](https://osu.ppy.sh/users/10325072)'s FC being able to bring back the lead, and thus the victory in a last-ditch effort.
+
+Nevertheless, both teams did not make it through the quarterfinals. The #23 seed from Sweden had an exceptional weekend, eliminating both Hong Kong and France from the tournament.
+
+### ::{ flag=PL }:: Poland vs ::{ flag=AU }:: Australia
+
+[**View the VOD here!**](https://www.twitch.tv/videos/1975435768)
+
+::{ flag=PL }:: [maliszewski](https://osu.ppy.sh/users/12408961) and ::{ flag=AU }:: [mrekk](https://osu.ppy.sh/users/7562902), two of the greatest solo tournament players to ever grace this game were about to settle once again who could carry harder in a 4v4 setting. It was the Australian roster getting the 3–0 head start with an unbeatable 4.3 million team score on [Umeri - paranoia](https://osu.ppy.sh/beatmapsets/2085311#osu/4368554), meanwhile the Polish team were just barely able to win their own picks for a 2–4 score. In a slightly worse case for them, this could have been a clean sweep. Still, Australia clutched out match point on the popular track [wowaka - Ura-Omote Lovers feat. Hatsune Miku](https://osu.ppy.sh/beatmapsets/1872527#osu/3853099).
```

Comment:
```suggestion
::{ flag=PL }:: [maliszewski](https://osu.ppy.sh/users/12408961) and ::{ flag=AU }:: [mrekk](https://osu.ppy.sh/users/7562902), two of the greatest solo tournament players to ever grace this game, were about to settle once again who could carry harder in a 4v4 setting. It was the Australian roster getting the 3–0 head start with an unbeatable 4.3 million team score on [Umeri - paranoia](https://osu.ppy.sh/beatmapsets/2085311#osu/4368554, while the Polish team were just barely able to win their own picks for a 2–4 score. In a slightly worse case for them, this could have been a clean sweep. Still, Australia clutched out match point on the popular track [wowaka - Ura-Omote Lovers feat. Hatsune Miku](https://osu.ppy.sh/beatmapsets/1872527#osu/3853099).
```